README.md

```diff
@@ -51,16 +49,33 @@ cap staging deploy # deploys development branch to staging
 
 Testing
 ------------------
-### Run All Tests
+### Database Configuration
+```bash
+rake db:create RAILS_ENV=test
+rake db:migrate RAILS_ENV=test
+rake db:seed RAILS_ENV=test
+```
 
+### Run Tests
 
 ```bash
 rake ci
 ```
 
-### Development
-While in development, you run tests without switching to the test solr instance:
+### Run Tests Separately
+
+```bash
+rake server:test
+```
+
+Then, in another terminal window:
+
+```bash
+rake spec
+```
+
+To run a specific test:
 
 ```bash
-rspec spec
+rake spec SPEC=spec/models/solr_document_spec.rb:27
```

Comment:
It was meant to be an example. I thought it was better than:

`rake spec SPEC=path/to/your/spec.rb:linenumber`

But maybe not. Actually, that doesn't seem that bad.